docs/pages/management/admin/troubleshooting.mdx

```diff
@@ -142,7 +142,8 @@ Teleport v9.0.4 git: go1.18
 
 <Tabs>
 <TabItem scope={["cloud", "enterprise","team"]} label="Commercial Teleport Editions">
-If you need help, please ask on our [community forum](https://github.com/gravitational/teleport/discussions). You can also open an [issue on GitHub](https://github.com/gravitational/teleport/issues) or create a ticket through your [Teleport account](https://teleport.sh).
+If you have a question or need assistance please submit a request 
+through the [Teleport support site](https://support.goteleport.com).
```

Comment:
I'd say _page_ or _portal_ instead of site, but maybe that's just personal preference.